.github/workflows/rust-mirai.yml

```diff
@@ -0,0 +1,64 @@
+name: Rust
+
+on:
+  pull_request:
+    types: ['opened', 'edited', 'reopened', 'synchronize']
+
+jobs:
+  install_and_run_MIRAI:
+
```

Comment:
Remove blank line

Reply:
these two blank lines are actually part of github's templates: https://github.com/actions/starter-workflows/blob/master/ci/rust.yml